src/components/Pagination/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, {useMemo} from 'react';
+import {useSearchParams} from 'react-router-dom';
+import {Pagination as PaginationRounded, Stack} from '@mui/material';
+
+interface IProps {
+    totalCount: number;
+}
+
+export const Pagination = ({totalCount = 1}: IProps) => {
+    const [searchParams, setSearchParams] = useSearchParams();
+    const currentPage = useMemo(() => Number(searchParams.get('page') || 1), [searchParams]);
+
+    const handlerNavigation = (event: React.ChangeEvent<unknown>, value: number) => {
```

Comment:
Nice to change **unknown** to somth appropriate

Reply:
According to Signature of onChange event of component <Pagination/> from Material UI docs:
**function(event: React.ChangeEvent<unknown>, page: number) => void
event: The event source of the callback.
page: The page selected.**

Perhaps it described with purpose that variable "event" doesn't use in this handler function